world-bank-app/src/App.js

```diff
@@ -43,9 +44,10 @@ function App() {
   }
 
   return (
-    <TwoCountriesOneIndicator
+    <TwoCountriesOneIndicatorOneYear
       countries={["Afghanistan", "Albania"]}
       indicator={"Debt service on external debt, long-term (TDS, current US$)"}
+      year={2006}
```

Comment:
I would remove creating the component here as it will be done in page-selection instead, avoids conflicts